src/mono/mono/metadata/assembly-load-context.c

```diff
@@ -101,6 +144,20 @@ ves_icall_System_Runtime_Loader_AssemblyLoadContext_InternalInitializeNativeALC
 	return alc;
 }
 
+void
+ves_icall_System_Runtime_Loader_AssemblyLoadContext_PrepareForAssemblyLoadContextRelease (gpointer alc_pointer, gpointer strong_gchandle_ptr, MonoError *error)
+{
+	uint32_t strong_gchandle = (uint32_t)GPOINTER_TO_UINT (strong_gchandle_ptr);
```

Comment:
nit: `MonoGCHandle`, not `uint32_t`.  Since https://github.com/mono/mono/pull/19039 we use pointer-sized gc handles now.